config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics.yaml.tpl

```diff
@@ -274,4 +274,61 @@ periodics:
     testgrid-tab-name: capi-e2e-conformance-ci-latest-{{ ReplaceAll $.branch "." "-" }}
```

Comment:
Let's rebase onto the latest commit on the master branch (so that the prowjob gen file is up-to-date)